pkg/testing/ig/ig.go

```diff
@@ -0,0 +1,244 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ig
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"syscall"
+
+	"github.com/blang/semver"
+)
+
+type IG struct {
+	path    string
+	image   string
+	version semver.Version
```

Comment:
Not yet.

Reply:
Should I remove the version logic since we aren't using it (the ig features and flags used are already supported by the last three versions, so should we check for the version before using them?)

Reply:
Should I remove the version logic? (since most of the features and flags are supported from the last five or six versions ig, and since we mostly install it via the repo itself, the version comes out to be 0.0.0, so it would give an error)